addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -427,16 +427,19 @@
             </span>
         </div>
 
-        <div class="d-flex flex-row">
```

Comment:
useless diff.
Also, not sure it does change something to move both Free Preview tags.
Or maybe I'm missing something.